### PR TITLE
Follow up of #158: Disabled unclassified marker.

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
This should fix up with the last remainder in #312.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,6 +53,12 @@ <h1>Wo ist Markt in <span class="nowrap"><select id="dropDownCitySelection"></se
 		  </div>
                   <input type="radio" name="display" id="other" value="other"><span>alle</span>
 		</label>
+		<label class="disabled">
```
Comment (k-nut) [inline index.html]:
```suggestion
		<label class="disabled" title="Die Öffnungszeiten dieser Märkte konnten nicht zugeordnet werden.">
```

**Comment by k-nut** [comment]
Thank you! I have added one small suggestion to somehow let users (at least desktop users that is) know what we mean with "sonstige". Let me know what you think about that.

**Comment by Ryuno-Ki** [comment]
Amended.

**Comment by k-nut** [comment]
Thank you again!

**Pull request merged**